src/main/java/bot/Storage.java

```diff
@@ -0,0 +1,82 @@
+package main.java.bot;
+
+import main.java.task.Task;

```

Comment:
Consider reordering this import statement (and others like it) to the bottom. Import statement order should be consistent.

Reply:
Nice catch - missed that. Thanks!